zanata-common-api/src/main/java/org/zanata/rest/client/IFileResource.java

```diff
@@ -57,6 +61,14 @@
             ClientResponse<String> acceptedFileTypes();
 
     @Override
+    @GET
+    @Path(ACCEPTED_TYPE_LIST_RESOURCE)
+    @Produces({MediaType.APPLICATION_JSON})
+    // /file/accepted_type_list
```

Comment:
I think this should go 2 lines up, so it is above `@Path(...)` (that is what is is about, right?)